apps/ng-web/src/app/eager/eager.component.ts

```diff
@@ -1,10 +1,17 @@
 import { Component } from '@angular/core';
-import { Actions } from '@ngneat/effects-ng';
+import {
+  Actions,
+  provideDirectiveEffects,
+  EffectsDirective,
+} from '@ngneat/effects-ng';
 import { eagerActionOne } from './state/eager.actions';
+import { EagerEffects } from './state/eager.effects';
 
 @Component({
   selector: 'effects-root',
   template: ` <div>Eager component loaded</div> `,
+  providers: [provideDirectiveEffects(EagerEffects)],
```

Comment:
Let's use it also in a lazy loaded component, just to not have any surprises :)

Reply:
I hope we've done with this feature. The more we go the more I regret😄

Reply:
lol. That's a legit and useful feature.

Reply:
This's the only thing that warms my heart. Everything about this feature is tough😄.